App.js

```diff
@@ -77,6 +83,42 @@ class App extends Component {
     LayoutAnimation.spring();
   }
 
+  getCurrentRouteName(navigationState) {
+    if (!navigationState) {
+      return null;
+    }
+    const route = navigationState.routes[navigationState.index];
+
+    if (route.routes) {
+      return this.getCurrentRouteName(route);
+    }
+
+    return route.routeName;
+  }
+
+  statusBarHandler(prev, next) {
+    const routeName = this.getCurrentRouteName(next);
+    const lightContentScreens = [
+      'MyProfile',
+      'Profile',
+      'Organization',
+      'Repository',
+      'Login',
+    ];
+
+    if (lightContentScreens.includes(routeName)) {
+      StatusBar.setTranslucent(true);
+      StatusBar.setBackgroundColor(
+        routeName === 'Login' ? colors.transparent : colors.black
```

Comment:
Can you replace `colors.black` by #262626. This would Android look like the iPhone:

Before:
<img width="338" alt="screen shot 2017-11-21 at 3 58 04 pm" src="https://user-images.githubusercontent.com/304450/33082726-bd635cba-cedd-11e7-80c2-a765dd67ff49.png">

After:
<img width="336" alt="screen shot 2017-11-21 at 4 49 17 pm" src="https://user-images.githubusercontent.com/304450/33082722-bca27a40-cedd-11e7-9e7a-6d0977a7d143.png">

Reply:
We should make a new color in config for this. 
The existant dark gray on profile pages is already #262626 but created with a rgba() thingy (that seems unneeded)